test/c3/appframework/appframework_test.go

```diff
@@ -211,4 +212,83 @@ var _ = Describe("c3appfw test", func() {
 			testenv.VerifyAppsCopied(deployment, testenvInstance, testenvInstance.GetName(), podNames, appListV1, false, true)
 		})
 	})
+
+	Context("Clustered deployment (C3 - clustered indexer, search head cluster)", func() {
+		It("c3, appframework: can deploy a C3 SVA and have ES app installed on SHC", func() {
+
+			// ES is a huge file, we configure it here rather than in BeforeSuite/BeforeEach to save time for other tests
+			// Upload ES app to S3
+			appListV1 = testenv.ESApp
```

Comment:
We are storing name of `ES App` in 2 different places. This can be simplified by just calling GetAppFileList with the app name.